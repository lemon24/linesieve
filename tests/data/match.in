match '([ab])_'
---
a_b
b_c
c_a
