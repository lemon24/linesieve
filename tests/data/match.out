a_b
b_c
